protobuf/decode.go

```diff
@@ -0,0 +1,254 @@
+package protobuf
+
+import (
+	"errors"
+	"fmt"
+	"math/big"
+
+	"github.com/Fantom-foundation/Substate/substate"
+	"github.com/Fantom-foundation/Substate/types"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/syndtr/goleveldb/leveldb"
+)
+
+type dbGetCode = func(types.Hash) ([]byte, error)
```

Comment:
I like to name type fucs with `Func` at the end. Also I think this does not need to be only `db` related func.
I suggest `dbGetCode` -> `getCodeFunc`

Reply:
`dbGetCode` -> `getCodeFunc`
[a660d9b](https://github.com/Fantom-foundation/Substate/pull/85/commits/a660d9b6b0a5a8bd3fd9548318927637bee943a4)